bin/common.sh

```diff
@@ -6,23 +6,36 @@ readonly clustersFile="$ENV_DIR/env/clusters.yaml"
 readonly helmfileOutputHide="(^\W+$|skipping|basePath=|Decrypting)"
 readonly helmfileOutputHideTpl="(^[\W^-]+$|skipping|basePath=|Decrypting)"
 readonly replacePathsPattern="s@../env@${ENV_DIR}@g"
+readonly yqDockerImage="mikefarah/yq:3"
+
+function dyq() {
```

Comment:
You can shadow yq and use command inside, e.g.:
```
function yq {
 # use command to ignore functions/aliases
  echo "yq wrapper"
  command yq "$1"
}
```

In this way, you don't need to change any `yq` call

Reply:
Neat! Will change